app/javascript/components/widgets/weather/moon-phase.jsx

```diff
@@ -155,7 +155,7 @@ MoonPhase.propTypes = {
   radius: PropTypes.string.isRequired,
   phase: PropTypes.string.isRequired,
   arcPath: PropTypes.string.isRequired,
-  arcAnimation: PropTypes.string.isRequired,
+  arcAnimation: PropTypes.objectOf(PropTypes.any).isRequired,
```

Comment:
if this is indeed an object now, we should define its shape here.

Reply:
Mistype there, it is an instance of the [Keyframes](https://styled-components.com/docs/api#keyframes) model now. The type is not exposed by the styled-components package, however. 

That line should really be `object` or `any`, but the linter advises strongly against such.

Any recommendations?